Assets/MRTK/Services/InputSimulation/SimulatedGestureHand.cs

```diff
@@ -98,14 +103,15 @@ private void EnsureProfileSettings()
             }
         }
 
+
         /// <summary>
         /// The GGV default interactions.
         /// </summary>
         /// <remarks>A single interaction mapping works for both left and right controllers.</remarks>
         public override MixedRealityInteractionMapping[] DefaultInteractions => new[]
         {
-            new MixedRealityInteractionMapping(0, "Select", AxisType.Digital, DeviceInputType.Select),
-            new MixedRealityInteractionMapping(1, "Grip Pose", AxisType.SixDof, DeviceInputType.SpatialGrip),
+            new MixedRealityInteractionMapping(0, "Select", AxisType.Digital, DeviceInputType.Select, selectAction),
+            new MixedRealityInteractionMapping(1, "Grip Pose", AxisType.SixDof, DeviceInputType.SpatialGrip, holdAction),
```

Comment:
holdAction doesn't seem like the appropriate name here. GripPoseAction maybe?

Reply:
Modified to properly use input profiles